scripts/upload-release-steps.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+set -euo pipefail
+
+# Generates and uploads pipeline steps for the experimental, beta and stable release
+
+trigger_step() {
+  local name="$1"
+  local trigger_pipeline="$2"
+
+  cat <<YAML
+  - name: ":rocket: ${name}"
+    trigger: "${trigger_pipeline}"
+    async: false
+    branches: "master show-version-in-block-steps"
+    build:
+      message: "Release for ${agent_version}, build ${build_version}"
+      commit: "${BUILDKITE_COMMIT}"
+      branch: "${BUILDKITE_BRANCH}"
+      meta_data:
+        agent-artifacts-build: "${BUILDKITE_BUILD_ID}"
+        agent-version: "${agent_version}"
+        agent-version-build: "${build_version}"
+        agent-version-full:  "${full_agent_version}"
+      env:
+        DRY_RUN: "${DRY_RUN:-false}"
+YAML
+}
+
+block_step() {
+  local label="$1"
+  cat <<YAML
+  - wait
+  - block: "${label}"
+YAML
+}
+
+output_steps_yaml() {
+  echo "steps:"
+
+  trigger_step \
+    "edge ${agent_version}.${build_version}" \
+    "agent-release-experimental"
+
+  block_step "beta?"
```

Comment:
Is there any way of adding a pop-up confirmation to the block step with some more description, like "This will release version v3.0.0-beta.23 and publish it to all customers" or something? Or can we only do pop-ups with fields? Might just be nice to be very explicit with verification. Especially for stable.